apps/src/studio/studio.js

```diff
@@ -1078,6 +1078,7 @@ Studio.init = function(config) {
   // Disable "show code" button in feedback dialog when workspace is hidden
   config.enableShowCode = !config.level.embed && studioApp.editCode;
   config.varsInGlobals = true;
+  config.generateFunctionPassBlocks = !!config.level.generateFunctionPassBlocks;
```

Comment:
It's newish to me too, though I think there are a few places where it's used in our code. I'm still not certain that I love it.